serverless.yml.example

```diff
@@ -103,7 +134,7 @@ resources:
       Properties:
         Endpoint: !GetAtt ingestQueue.Arn
         Protocol: sqs
-        Region: "#{AWS::Region}"
+        Region: "${aws:region}"
```

Comment:
- #293

Reply:
this is now a features of serverless 3, whereas before it was in the pseudo-parameters plugin.